config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :tasks
+  # The following is included in resources
+  # get '/tasks', to: 'tasks#index', as: 'tasks'
+  # get '/tasks/new', to: 'tasks#new', as: 'new_task'
+  # post '/tasks', to: 'tasks#create'
+  # get '/tasks/:id', to: 'tasks#show', as: 'task'
+  # get '/tasks/:id/edit', to: 'tasks#edit', as: 'edit_task'
+  # patch '/tasks/:id', to: 'tasks#update'
+  # delete '/tasks/:id', to: 'tasks#destroy'
```

Comment:
A good note, but in the future, delete this! :) If you ever forget what routes you have `rails routes` is your friend